templates/pipeline-template-releasebranch.yaml

```diff
@@ -0,0 +1,95 @@
+parameters:
+- name: 'terraform'
+  type: object
+  default:
+    version: '0.15.4'
+    modules_repo: 'https://dev.azure.com/skfdc/REP-SW/_git/terraform-modules'
+    cache_dir: '.terraform.d/plugin-cache'
+- name: 'freezeQueryId'
+  type: string
+  default: 'fde6db1b-dec1-4dc5-a553-0a047ef96864'
+- name: 'buildJobs'
+  type: jobList
+- name: project
+  type: string
+
+stages:
```

Comment:
I notice that you don't have the if statement checking for if it's a main branch or not.

How are you thinking this is to be used in the FF pipeline? What I am thinking is either:
1. The template knows which branch should do what and you just import a 'smart/generic' template
2. The pipeline keeps track of which template it should use depending on what branch it is on

I kinda prefer option 2, making templates modular and interchangeable.

Reply:
Option 1 is just having one template and just condition the sandbox part depending on if you are on main or not
Might be the simplest solution right now

Reply:
I was thinking that in the build yaml you change what template you want to use then change back to old one before merging.

Reply:
I would really prefer not having to do that :sweat_smile: . I think we need to decide weather we want to use this for FF och swap it for the regular way of doing it.

If we are going to keep this, I would like to have an branch pipeline which doesn't require manual swapping and committing.

Reply:
I feel that is a bigger thing than this PR. This simply adding a template that can release any branch.